egs/wsj/s5/steps/nnet3/get_egs_raw_nnet.sh

```diff
@@ -0,0 +1,411 @@
+#!/bin/bash
```

Comment:
I don't like the name of this program, because there is no inherent connection with a 'raw' neural net- the get_egs program doesn't need to see the neural net.  maybe call it get_egs_targets.sh, as in this case you supply more general targets, not alignments.